app/models/goal.py

```diff
@@ -4,3 +4,15 @@
 
 class Goal(db.Model):
     goal_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    __tablename__ = "goals"
+    tasks = db.relationship("Task", backref="goal", lazy=True)
+
+
+
+    def json_2(self):
```

Comment:
Nice helper function!

Reply:
I am curious about why you named your function with a `2`. What does the `2` represent?

Reply:
Good question! Made sense at the time. However, now that am more familiar with building these helper functions I can have a better name for them. :)